.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     matrix:
       linux:
         CONFIG: azure-linux-64-comp7
-        IMAGE_NAME: condaforge/linux-anvil-comp7
+        IMAGE_NAME: condaforge/linux-anvil-cuda:10.1
```

Comment:
Can you revert this, so that we can merge this?

Reply:
done